plugin-server/src/utils/db/db.ts

```diff
@@ -562,7 +562,9 @@ export class DB {
         RETURNING version`
 
         const updateResult: QueryResult = await this.postgresQuery(queryString, values, 'updatePerson', client)
-
+        if (updateResult.rows.length == 0) {
+            throw new Error(`Person with team_id="${person.team_id}" and uuid="${person.uuid} couldn't be updated`)
```

Comment:
do we have any idea why? is it a race condition thing? if so, lets throw a `RaceConditionError`

Reply:
I explained in the summary already:

> This is a problem we've had in the old update path for a long time, since we don't lock the person row the person could have been merged or deleted before we reach the update.
This is an improvement as we no longer send the outdated person to ClickHouse and this could have created some problems there with duplicate persons in the past.